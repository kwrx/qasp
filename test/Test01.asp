c(1..5).
d(1..5).

%@exists
c(6).
in_a(X) | out_a(X) :- b(X), c(X).
b(X) :- d(X).
%@foreach
d(X) :- c(X).
%@constraints
:- in_a(X), X > 3.