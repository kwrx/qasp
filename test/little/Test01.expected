QASP 1.1

{b}
